src/platform/datapath_raw_xdp_win.c

```diff
@@ -532,6 +431,23 @@ CxPlatDpRawInterfaceInitialize(
         CxPlatDatapathSqeInitialize(&Queue->TxIoSqe.DatapathSqe, CXPLAT_CQE_TYPE_SOCKET_IO);
         Queue->TxIoSqe.IoType = DATAPATH_XDP_IO_SEND;
 
+        uint32_t QueueIndex = UINT32_MAX;
+        for (uint32_t j = 0; j < Interface->QueueCount; j++) {
+            if (Processors[j] == i) {
+                QueueIndex = j;
+                break;
+            }
+        }
+        if (QueueIndex == UINT32_MAX) {
+            Status = QUIC_STATUS_INVALID_STATE;
+            QuicTraceEvent(
```

Comment:
@mtfriesen I'm hitting this error in test (definitely over duonic, but also netperf is failing, but I don't have logs). I'm assuming that means we're trying to bind a queue to a processor that doesn't have an RSS queue? I'm trying to decide how to handle this. Thoughts?

Reply:
More queues than processors in the indirection table is completely legal. I'd just leave any leftover queues unaffinitized and hope for the best.

Reply:
Hmm, ok, I'm not that familiar with this QUIC code. NICs just indicate packets on the processors in the indirection table - the rest are idle.